examples/metrics-basic/src/main.rs

```diff
@@ -140,6 +140,32 @@ async fn main() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
     // Metrics are exported by default every 30 seconds when using stdout exporter,
     // however shutting down the MeterProvider here instantly flushes
     // the metrics, instead of waiting for the 30 sec interval.
-    meter_provider.shutdown()?;
-    Ok(())
+    let shutdown_result = meter_provider.shutdown();
```

Comment:
@scottgerring @lalitb @TommyCpp I put this just to think out loudly, showcasing what a user can possibly do with shutdown result from MeterProvider. (reader/exporter is not directly used by end users)